edges/industry/industry_final_demand_for_chemical_hydrogen_non_energetic-industry_final_demand_for_chemical_fertilizers_hydrogen_non_energetic@hydrogen.ad

```diff
@@ -1,2 +1,5 @@
 - type = share
 - reversed = false
+- groups = [chemical_industry]
+
+#~ parent_share = SHARE(industry_final_demand_for_chemical_hydrogen_non_energetic_parent_share, industry_final_demand_for_chemical_fertilizers_hydrogen_non_energetic)
```

Comment:
Is it correct that this line is commented out?

Reply:
Ah good point! I first thought that I had to add these shares, but in the end they did not seem to be needed (since demand is zero in the start year). So they can be deleted

Reply:
The parent_shares are needed for some datasets! So the shares are added to all datasets (https://github.com/quintel/etsource/pull/2041/commits/543847a8b00b6316fcea45e069bf2c580ecb8ec6)
![image (5)](https://user-images.githubusercontent.com/32833996/58885292-3ca9c280-86e2-11e9-888a-44cf0e12473f.png)